.github/workflows/ant.yaml

```diff
@@ -44,19 +44,59 @@ jobs:
       - run: yarn aos:build
       - run: yarn test
 
-  notify:
+  publish:
     runs-on: ubuntu-latest
     needs: [unit, integration]
+    if: github.ref == 'refs/heads/main'
+    outputs:
+      publish-output: ${{ steps.set-publish-output.outputs.publish-output }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4.0.2
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+      - run: yarn --frozen-lockfile
+      - run: yarn aos:build
+        id: publish-lua-code
+      - run: yarn aos:publish
+        env:
+          ARWEAVE_PUBLISHING_KEY: ${{ secrets.ARWEAVE_PUBLISHING_KEY }}
```

Comment:
nit: i think `JWK` is fine? it's what we use most everywhere else. i've added it to the repo

Reply:
went with WALLET